python-package/lightgbm/basic.py

```diff
@@ -1,5 +1,6 @@
 # coding: utf-8
```

Comment:
Please add `Sequence` here
https://github.com/microsoft/LightGBM/blob/c7134fa7ccbe1a1b1819ac2a66407504d3fab27a/python-package/lightgbm/basic.py#L1944

Reply:
Since we are going to support `Sequence` in follow-up PR, I'm going to mark this as resoved for now.